_pages/en/it-picture-long-term.md

```diff
@@ -59,6 +60,12 @@ Said service representatives would be able to...
 GC departments have common kiosks to offer services, such as passports, taxes, grants as well as benefits from ESDC.
 Outreach programs are deployed to raise awareness in low uptake areas.
 
+### Private Sector
+
+- Private sector includes for profit companies, not for profit organizations, academia, and private citizens
```

Comment:
Not sure what this is intended to mean...?

Reply:
To clarify that private sector isn't limited to private companies. But that we will work with non profit, academia, and even private citizens

Reply:
See commit #02a1519 with rewording